homeassistant/docker-compose.homeassistant.yml

```diff
@@ -0,0 +1,26 @@
+version: '2'
+
+services:
+  homeassistant:
+    image: 'ghcr.io/home-assistant/home-assistant:${HOME_ASSISTANT_IMAGE_VERSION:-stable}'
+    # devices: # For passing through USB, serial or gpio devices.
+      # - '/dev/ttyUSB0:/dev/ttyUSB0'
+    environment:
+      GUID: 1000
+      PUID: 1000
+    healthcheck:
+      test: ['CMD', 'curl', '0.0.0.0:8123']
```

Comment:
**Suggestion:** The healthcheck command is using '0.0.0.0' as the host, which might not work in all environments. Consider using the service name 'homeassistant' instead.
```suggestion
    healthcheck:
      test: ['CMD', 'curl', 'homeassistant:8123']
```